src/current/v23.1/debezium.md

```diff
@@ -0,0 +1,92 @@
+
+- Ensure you have a secure, [publicly available]({% link cockroachcloud/network-authorization.md %}) CockroachDB cluster running the latest **{{ page.version.version }}** [production release](https://www.cockroachlabs.com/docs/releases/{{ page.version.version }}), and have created a [SQL user]({% link {{ page.version.version }}/security-reference/authorization.md %}#sql-users).
+- Have [Debezium](https://debezium.io/), [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.html), and [Kafka](https://kafka.apache.org/) already set up. This documentation assumes you have already added data from your [source database](https://debezium.io/documentation/reference/stable/connectors/index.html) to a Kafka topic.
+
+## Migrate data to CockroachDB
+
+Once all of the [prerequisite steps](#before-you-begin) are completed, you can use Debezium to migrate data to CockroachDB.
+
+1. To write data from Kafka to CockroachDB, use the Confluent JDBC Sink Connector. First use the following `dockerfile` to create a custom image with the JDBC driver:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    FROM quay.io/debezium/connect:latest
+    ENV KAFKA_CONNECT_JDBC_DIR=$KAFKA_CONNECT_PLUGINS_DIR/kafka-connect-jdbc \
+
+
+    ARG POSTGRES_VERSION=latest
+    ARG KAFKA_JDBC_VERSION=latest
+
+
+    # Deploy PostgreSQL JDBC Driver
+    RUN cd /kafka/libs && curl -sO https://jdbc.postgresql.org/download/postgresql-$POSTGRES_VERSION.jar
+
+
+    # Deploy Kafka Connect JDBC
+    RUN mkdir $KAFKA_CONNECT_JDBC_DIR && cd $KAFKA_CONNECT_JDBC_DIR &&\
+       curl -sO https://packages.confluent.io/maven/io/confluent/kafka-connect-jdbc/$KAFKA_JDBC_VERSION/kafka-connect-jdbc-$KAFKA_JDBC_VERSION.jar
+    ~~~
+
+1. Create the JSON configuration file needed to create the sink:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    {
+       "name": "pg-sink",
+       "config": {
+           "connector.class": "io.confluent.connect.jdbc.JdbcSinkConnector", 
+           "tasks.max": "10",
+           "topics" : "{topic.example.table}",
+           "connection.url": "jdbc:postgresql://{host}:{port}/{user}?sslmode=require",
+           "connection.user": "{username}",
+           "connection.password": "{password}",
+           "insert.mode": "upsert",
+           "pk.mode": "record_value",
+           "pk.fields":"id",
+           "database.time_zone": "UTC",
+           "auto.create":true,
+           "auto.evolve": false,
+           "transforms": "unwrap",
+           "transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState"
+       }
+    }
+    ~~~
+    
+    Specify the **Connection URL** in [JDBC format]({% link {{ page.version.version }}/connect-to-the-database.md %}?filters=java&#step-5-connect-to-the-cluster). For information about where to find the CockroachDB connection parameters, see [Connect to a CockroachDB Cluster]({% link {{ page.version.version }}/connect-to-the-database.md %}).
+
```

Comment:
Wait, are we missing a step here? How do you actually create the sink using this config file?

Reply:
That's a good point. We could add the step explicitly to POST this JSON to the `/connectors` endpoint for Kafka Connect and provide this reference documentation as a step 3 https://kafka.apache.org/documentation/#connect_rest

I'd stop there though because from that point it should be handled on the Kafka side of things.